server/src/main/java/org/solarbank/server/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,69 @@
+package org.solarbank.server;
+
+import java.util.List;
+import org.solarbank.server.ErrorMessage;
+import org.solarbank.server.dto.ErrorResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ControllerExceptionHandler {
+
+    private void log(Exception ex) {
+        ex.printStackTrace();
+    }
+
+    @ExceptionHandler
+    public ResponseEntity<ErrorResponse> handleValidationException(
+            MethodArgumentNotValidException ex) {
+
+        StringBuilder message = new StringBuilder();
+        BindingResult bindingResult = null;
+
+        try {
```

Comment:
should catch any exceptions with getBindingResult() and getFieldErrors() methods, also handles if FieldErrors empty